releases/v3.0.0.toml

```diff
@@ -0,0 +1,2930 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "registry"
+github_repo = "distribution/distribution"
+
+# previous release
+previous = "v2.8.3"
+
+pre_release = false
+
+preface = """\
+Welcome to the 3.0.0 release of registry!
+
+This is the first major release in years! It's an accumulation of effort
+that's bringing major improvements in performance, features, security and
+general code quality.
+
+See the changelog below for full list of changes.
+
+### Highlights
+
+* Support for Image Manifest v2 Schema v1 has been deprecated
+* Support for `docker/libtrust` has been deprecated in favour or `go-jose/go-jose`
+* The only supported storage drivers are: S3, GCS and Azure
+* Major performace improvements across all storage drivers
+* Major dependencies update (see the full list below)
+* Online documentation is available on https://distribution.github.io/distribution/
```

Comment:
Before GA we should mention somewhere that the go module was renamed (docker -> distribution, as well as renamed to "v3")